docs/developer/payments/lifecycle.mdx

```diff
@@ -56,107 +56,193 @@ Creating the refund will change the status to:
-The status takes into account the following fields: `authorizedAmount`, `chargedAmount`, `authorizePendingAmount`, and `chargePendingAmount`:
+The status is calculated based on summing up all transactions assigned to the `Checkout` (`authorizedAmount`, `authorizationPendingAmount`, `chargedAmount`, `chargePendingAmount` on `TransactionItem`) and by comparing it with the `checkout.totalPrice` value.
 
-- **NONE**: No funds have been authorized.
-- **PARTIAL**: The available funds do not fully cover the checkout's total amount.
-- **FULL**: The funds fully cover the checkout's total amount. At this point, `checkoutComplete` can be called to convert the checkout into an order. 
+It uses a formula:
+
+```
+AmountToCover = checkout.totalPrice
+CoveredAmount = sum(checkout.transactions.chargedAmount) + sum(checkout.transactions.chargePendingAmount) + sum(checkout.transactions.authorizePendingAmount) + sum(checkout.transactions.authorizedAmount)
+```
+
+Based on that calculation `Checkout.authorizeStatus` can have the following values:
+
+- **`NONE`**: No funds have been authorized. (`CoveredAmount <= 0`)
+- **`PARTIAL`**: Some funds were authorized, but they do not cover required amount (`AmountToCover > CoveredAmount`)
+- **`FULL`**: The authorized funds fully cover the required amount. (`AmountToCover <= CoveredAmount`)
 
 :::caution
-Note that pending amounts (success or failure) will be confirmed later by the payment app. This means the system can create an order that may remain unpaid if the pending amounts eventually fail.
+Transaction that is pending will be confirmed later by the payment app (it will result in either success or failure).
+
+This means user can create an Order that may remain unpaid if the pending amounts eventually fail.
 :::
 
 #### Available transitions
 
 - Status is updated when Checkout's `totalAmount` is changed.
 - Status is updated when Checkout's `TransactionItem`'s amounts are updated.
 
-#### Authorization status calculation for Order
+### Authorization status calculation for Order
+
+[`Order.authorizeStatus`](/api-reference/orders/objects/order#orderauthorizestatusorderauthorizestatusenum---) status depends on:
 
-The status is calculated based on all transactions assigned to the `Order`.
-Authorization means that funds have been either fully or partially authorized or charged.
-It uses the fields `authorizedAmount` and `chargedAmount`, and the calculation is adjusted by subtracting `order.totalGrantedRefund` from the total.
+- sum of **charged** transactions (`TransactionItem.chargedAmount`)
+- sum of **authorized** transactions (`TransactionItem.authorizedAmount`)
+- `order.totalGrantedRefund` value
+- `order.total` value
 
 :::caution
-Unlike checkout, this status does not consider pending amounts.
+Unlike Checkout, this status does not take into consideration pending transactions.
```

Comment:
Maybe we should mention here that it might result in situation that order completed from a  checkout with `authorizationStatus: FULL` will have `authorizationStatus: NONE`